azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/CallingFragment.kt

```diff
@@ -181,4 +181,8 @@ internal class CallingFragment :
     private fun displayParticipantList() {
         viewModel.getParticipantListViewModel().displayParticipantList()
     }
+
+    private fun switchFloatingHeader() {
```

Comment:
Is function necessary?

Reply:
Adam, this function is used at multiple places. (2 times in this view)